ui/packages/catalog-ui-search/src/main/webapp/component/input/date/input-date.view.js

```diff
@@ -24,6 +24,8 @@ const Common = require('../../../js/Common.js')
 const user = require('../../singletons/user-instance.js')
 require('eonasdan-bootstrap-datetimepicker')
 
+const timeOnlyFormat = 'HH:mm:ss.SSS'
```

Comment:
This is only used inside `hasSameTime`, can probably be moved into that function.